### DISCUSSION TITLE
Integrate design tokens from Figma

### DISCUSSION THREAD

**Discussion by WilliamKelley** [prenda-school/prenda-spark#325] [Ideas]
Re: https://jansix.at/blog/creating-design-systems-based-on-design-tokens-figma-tokens

**Comment by alecdanz** [comment]
I'm happy to install this plugin and give it a try if we would like to attempt implementing these for our new set of colors.

**Reply by WilliamKelley** [reply]
Sure, we should pair on that because you'll likely need tokens that you don't have access to create (i think) and just general know how of GH branching and stuff.

It might be a while before the resulting tokens are _actually_ used in implementation, not just connected to the repo, but it's valuable exploration.
